gulp-tasks/reference-docs/build-js-docs.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const gutil = require('gulp-util');
+const fs = require('fs-extra');
+const path = require('path');
+const glob = require('glob');
+const wfHelper = require('../wfHelper');
+
+const canFindJSDocConf = (jsdocConfPath) => {
```

Comment:
Please document functions with JSDoc. Include at least a brief description, and `@param`/`@return` as appropriate.  Will still merge if not added.